src/localization/english.ts

```diff
@@ -18,10 +18,10 @@ export var englishStrings = {
   completingSurvey: "Thank you for completing the survey!",
   completingSurveyBefore:
     "Our records show that you have already completed this survey.",
-  loadingSurvey: "Loading Survey...",
+  loadingSurvey: "Loading survey...",
   optionsCaption: "Choose...",
   value: "value",
-  requiredError: "Please answer the question.",
+  requiredError: "Please answer this question.",
   requiredErrorInPanel: "Please answer at least one question.",
   requiredInAllRowsError: "Please answer questions in all rows.",
```

Comment:
'Please answer questions in all rows' What do you mean by all rows? If every row has a question, you could just say 'Please answer all the questions'.

Reply:
I did not change this string, but you are right. It should be changed.